Revert-and-Regression-Policy.md

```diff
@@ -23,72 +77,37 @@
 
--   Introduces a flaky (there is any reported failure of the test on the develop branch) or consistently failing test.
-    
--   Introduces a regression in a critical functionality.
-    
-
-Then, follow these steps:
-
-1.  Automatically revert the PR if it has no merge conflicts or merge conflicts are trivial to resolve and inform the author that you have reverted their PR. You can find instructions for reverting PRs on Github [here](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/reverting-a-pull-request).
-    
-2.  If the revert PR has non-trivial merge conflicts, do not revert. Create an issue and mark it as a blocking bug for the on-going release and inform the author of the PR about the same by @-mentioning on the PR thread.
-    
-3.  If the PR introduced a flaky or failing test and contains other changes too, create a PR to delete or disable the test. Also, create an issue to add back the deleted/disabled test and inform the author of the PR about the same by @-mentioning on the PR thread.
-    
-4.  If the functionality is broken on the production server and needs to be fixed, inform the [release coordinator](https://github.com/oppia/oppia/wiki/Release-Schedule#release-coordinators-and-qa-coordinators-for-upcoming-releases) for the previous release so that a hotfix can be done.
-    
-5.  Add "PR: require post-merge sync to HEAD" label to the revert or fix PR if developers need to update their branches once the fix/revert PR has been merged into develop. Oppiabot will comment on open PRs to merge from develop once the reverted PR has been merged.
+    Repeat this step until the tool reports the offending commit.
 
-## How to revert a PR?
+### How to revert a PR
 
-In the PR page, you will find a “revert” button at the point where the PR was merged.  
+In the PR page, you will find a “revert” button at the point where the PR was merged.
 ![](https://lh5.googleusercontent.com/dGgjIANi9zathEV_g9e5FKjpWTSn2tUSIUCdwalzUN6w1ocR1j5cuMoxq6tPOMYtk-1xsMPxj7tdkkK9jbOJP8f399DE1AAKmmCIcBMyYmd0MGJ3j3tO6P1R3b4frVMZy72UnAW_)
 
 Clicking on this button will create a new PR with the reverted changes. Feel free to add a detailed comment in the PR description explaining why the PR needs to be reverted and tag the author of the original PR.
 
 If you see this:
 
-![](https://lh3.googleusercontent.com/4uYWOO2yXW8eBVTG9BH-wYZKRo7rB2WNUBbBudtaprlp4btYJ3avdQP-fRnZSBXGAq1DM05Hc_c9haASvHTFF8gRHgPZNqQ3ZKZc7vkZrPy13rTZ2EwOuQXQM6Sz7j0gVpF-61Z5)  
-  
-You will need to make the code changes on your local machine and create a PR by hand.
-
-## What do I do if I caused a regression?
-
-If the PR that introduced the regression hasn't been reverted, please follow the steps above to revert. Once your PR is reverted, follow these steps:
+![](https://lh3.googleusercontent.com/4uYWOO2yXW8eBVTG9BH-wYZKRo7rB2WNUBbBudtaprlp4btYJ3avdQP-fRnZSBXGAq1DM05Hc_c9haASvHTFF8gRHgPZNqQ3ZKZc7vkZrPy13rTZ2EwOuQXQM6Sz7j0gVpF-61Z5)
```

Comment:
Where is this image stored?

Reply:
Someone's Google account. I've downloaded the images and checked them into git